enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,269 @@
+
+#### During Install
+
+1. The CEO will watch for new masters and the arbiter role
+2. CEO will create the operand for the etcd deployments that have tolerations
+   for the arbiter
+3. Operators that have tolerations for the arbiter should be scheduled on the
+   node
+4. The install should proceed as normal
+
+### API Extensions
+
+The `config.infrastructure.controlPlaneTopology` enum will be extended to
+include `ArbiterHighlyAvailable`
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+At the time being there is no impact on Hypershift since this edge deployment
+will require running the control plane.
+
+#### Standalone Clusters
+
+This change is relevant to standalone deployments of OpenShift at the edge or
+datacenters. This enhancement specifically deals with this type of deployment.
+
+#### Single-node Deployments or MicroShift
+
+This change does not effect Single Node or MicroShift.
+
+### Implementation Details/Notes/Constraints
+
+Currently there are some behavior unknowns, we will need to put out a POC to
+validate some of the desires in this proposal. In it's current version this
+proposal is not exhaustive but will be filled out as we implement these goals.
+
+We currently expect this feature to mainly be used by `baremetal` installs, or
+specialized hardware that is built to take advantage of this type of
+configuration. In the current design we make two paths for cloud and baremetal
+installs in the installer. However, the cloud install is primarily for testing,
+this might mean that we simplify the installer changes if we are the only ones
+using cloud installs, since we can simply alter the manifests in the pipeline
+with out needing to change the installer.
+
+### Risks and Mitigations
+
+The main risk in this enhancement is that because we are treating one of the
+master nodes in a 3 node cluster as an arbiter, we are explicitly evicting
+processes that would otherwise be a normal supported upstream configuration such
+as a compact cluster. We run the risk of new components being critical to HA not
+containing the proper tolerations for running on the arbiter node. One of the
+mitigations we can take against that is to make sure we are testing installs and
+updates.
+
+Another risk we run is customers using an arbiter node with improper disk speeds
+below that recommended for etcd, since etcd is sensitive to latency between
+members, we should provide proper guidance so that the arbiter node doesn't
+become a bottleneck for etcd.
+
+### Drawbacks
+
+A few drawbacks we have is that we will be creating a new variant of OpenShift
+that implements a new unique way of doing HA for kubernetes. This does mean an
+increase in the test matrix and all together a different type of tests since
+
+## Open Questions [optional]
+
+1. In the future it might be desired to add another master and convert to a
+   compact cluster, do we want to support changing ControlPlaneTopology field
+   after the fact?
```

Comment:
see above, this is already listed in the requirements and should be possible. Maybe not on initial release, but then on the next one.

Reply:
Feature Requirement number 8 states: 
`It must be possible to explicitly schedule additional workload to the arbiter node. That is important for 3d party solutions (e.g. storage provider) which also have  quorum based mechanisms.`
The use case is e.g. ODF in replica-2 mode, where some ODF components also need 3 deployments (the ceph mon for example). 
How is this supported by this ER? I think we should mention / describe this case.

Reply:
I guess that's not clear here, the idea is that anyone that wishes to run on an arbiter node will need to just have the proper tolerations or directly target the node. I'll capture that in the doc more explicitly.